src/List.js

```diff
@@ -77,7 +78,14 @@ export function List() {
     const timeToMinDate = new Date() - minDate + ONE_MINUTE;
     //re-render the page so the item unchecks when it should be unchecked
     setTimeout(() => setReRender({}), timeToMinDate);
-  }, [reRender, items]);
+  }, [reRender, items, ONE_MINUTE]);
```

Comment:
Should we `ONE_MINUTE` to `ONE_DAY` given that we're all done demo-ing this?

Reply:
updated!